packages/react-components/src/components/Input/Input.tsx

```diff
@@ -1,34 +1,90 @@
 import * as React from 'react';
 import cx from 'clsx';
 
+import {
+  VisibilityOn as VisibilityOnIcon,
+  VisibilityOff as VisibilityOffIcon,
+} from '@livechat/design-system-icons/react/material';
+
 import styles from './Input.module.scss';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
 
-type InputSize = 'xsmall' | 'small' | 'medium' | 'large';
+type InputSize = 'compact' | 'medium' | 'large';
```

Comment:
For me it could be a generic `size` type lifted up in folders structure as we are slowly starting to repeat ourselves 

![Screenshot 2022-11-15 at 08 42 54](https://user-images.githubusercontent.com/46934668/201858979-6e4b4272-a9ba-4b7c-a5c8-6296f58413b1.png)